ONSdigital/eq-survey-runner
# Remove map method from answer store

### What is the context of this PR?
From @ajmaddaford: "The Answer Store map method was originally created to output a WTForms POST data compatible format. This is then used to serialise the form data back into the WTForms form object. 

The map method has since been used in the summary context, accessing the answer values from this flattened list. Accessing the answers in this way is unclear and from https://github.com/ONSdigital/eq-survey-runner/pull/1158 will mean both the mapped answer values (used for answer values) and the answer store are being used."

Latest changes include adding method chaining to the answer store objects as well as reducing the overall complexity of building summary context.

### How to review 
* Complete survey and check summary page displays expected results
* Ensure changes are clear and obvious
* Check tests all pass across environments
